### DISCUSSION TITLE
Importing local .ICS file

### DISCUSSION THREAD

**Discussion by Y-a-n-i-n-a** [davish/obsidian-full-calendar#393] [Feature Requests]
For those of us with corporate calendars one workaround could be to export them into ICS and then import them. Thanks for considering this.

**Comment by macleod-ee** [comment]
I would assume that this would also let us have access to more custom recurring events, as we could create them in Thunderbird (as an example), export them to a directory in Obsidian, and have FC read from that source. Not sure of the background but could work for #22 as well.

**Comment by BrianBatchelder** [comment]
I'd like to have this to go with a shortcut I've been toying with that exports the next seven days of events (as .ics files) to the Events folder. Rather than importing them, however, I'd just like Full Calendar to parse and display them as is.

**Comment by jbaughman42** [comment]
I'd find this very useful, too. My particular use-case is trying to pull data from a corporate MS 365 Outlook/Exchange setup, and for security reasons I can't connect Obsidian directly to Outlook. The workaround would be to export a week's worth of appointments to .ics and import to Obsidian.
